docs/src/man/comparisons.md

```diff
@@ -1,77 +1,204 @@
+|                          | `df[['z']].agg(['mean'])`                             | `combine(df, :z => mean ∘ skipmissing)` |
+| Add new columns          | `df.assign(x_mean = df['x'].mean())`                  | `transform(df, :x => mean => :x_mean)`  |
+| Rename columns           | `df.rename(columns = {'x': 'x_new'})`                 | `rename(df, :x => :x_new)`              |
+| Pick & transform columns | `df.assign(x_mean = df['x'].mean())[['x_mean', 'y']]` | `select(df, :x => mean, :y)`            |
+| Sort rows                | `df.sort_values(by = ['x'])`                          | `sort(df, :x)`                          |
+
+Note that Julia propagates `missing` data by default for safety reasons. The `skipmissing` function
+can be used to remove missing data. See more details at the [Additional Differences](@ref) section below.
+
+### Grouping data and aggregation
+
+| Operation                    | pandas                                                                                    | DataFrames.jl                               |
+| :--------------------------- | :---------------------------------------------------------------------------------------- | :------------------------------------------ |
+| Aggregate by groups          | `df.groupby('grp')['x'].mean().reset_index()`                                             | `combine(groupby(df, :grp), :x => mean)`    |
+| Aggregate and add columns    | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')`                       | `transform(groupby(df, :grp), :x => mean)`  |
+| Aggregate and select columns | `df.join(df.groupby('grp')['x'].mean(), on='grp', rsuffix='_mean')[['grp','x_mean','y']]` | `select(groupby(df, :grp), :x => mean, :y)` |
+
+### More advanced commands
+
+| Operation                 | pandas                                                                       | DataFrames.jl                                                 |
+| :------------------------ | :--------------------------------------------------------------------------- | :------------------------------------------------------------ |
+| Complex Function          | `df[['z']].agg(lambda v: np.mean(np.cos(v)))`                                | `combine(df, :z => v -> mean(cos, skipmissing(v)))`           |
+| Transform several columns | `df.agg({'x': max, 'y': min})`                                               | `combine(df, :x => maximum, :y => minimum)`                   |
+|                           | `df[['x','y']].mean()`                                                       | `combine(df, [:x, :y] .=> mean)`                              |
+|                           | `df.filter(regex=("^x")).mean()`                                             | `combine(df, r"^x" .=> mean)`                                 |
+|                           | `df[['x', 'y']].agg([max, min])`                                             | `combine(df, ([:x, :y] .=> [maximum minimum])...)`            |
```

Comment:
same here, look at pandas output:
```
>>> df[['x', 'y']].agg([max, min])
     x  y
max  6  9
min  1  4
```
vs DataFrames.jl output:
```
julia> combine(df, ([:x, :y] .=> [maximum minimum])...)
1×4 DataFrame
│ Row │ x_maximum │ y_maximum │ x_minimum │ y_minimum │
│     │ Int64     │ Int64     │ Int64     │ Int64     │
├─────┼───────────┼───────────┼───────────┼───────────┤
│ 1   │ 6         │ 9         │ 1         │ 4         │
```

Reply:
It seems to be quite complex to arrive at the same result. Maybe a missing feature in DataFrames.jl?

```
julia> @pipe combine(df, ([:x, :y] .=> [maximum minimum])...) |>
           stack(_, All()) |>
           transform(_, :variable => ByRow(x -> split(get(x), "_")[2]) => :measure) |> 
           transform(_, :variable => ByRow(x -> split(get(x), "_")[1]) => :column) |>
           select(_, Not(:variable)) |>
           unstack(_, :column, :value)
2×3 DataFrame
│ Row │ measure  │ x      │ y      │
│     │ SubStri… │ Int64? │ Int64? │
├─────┼──────────┼────────┼────────┤
│ 1   │ maximum  │ 6      │ 9      │
│ 2   │ minimum  │ 1      │ 4      │
```

or 
```
julia> vcat(
         transform(
           combine(df, :x => minimum => :x, :y => minimum => :y),
                   1 => (x->"min") => :op),
         transform(
           combine(df, :x => maximum => :x, :y => maximum => :y),
                          1 => (x->"max") => :op))
2×3 DataFrame
│ Row │ x     │ y     │ op     │
│     │ Int64 │ Int64 │ String │
├─────┼───────┼───────┼────────┤
│ 1   │ 1     │ 4     │ min    │
│ 2   │ 6     │ 9     │ max    │
```

I think we have two options here:
1. Remove the example altogether
2. Add another note about the difference in shapes

I favor option 2. Please let me know what you think.  Thanks